packages/perspective-viewer-datagrid/src/js/event_handlers/keydown/edit_keydown.js

```diff
@@ -82,6 +84,17 @@ const moveSelection = lock(async function (
     }
 });
 
+function isLastCell(model, table, target) {
+    const meta = table.getMeta(target);
+    return meta.y === model._num_rows - 1 && meta.x === model._column_paths.length - 1;
```

Comment:
This checks both `x` _and_ `y`, e.g. the bottom-right corner - we only want to check the `y` parameter here, as per #2347